### PR TITLE
fix(components/ActionDropdown): fix possibility to use onSelect function

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

We can't define one onSelect function for current implementation of ActionDrodpown

**What is the new behavior?**

We can define onSelect once and don't specify onClick for every MenuItem

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://322.talend.surge.sh)

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js [hunk 1]
```diff
@@ -49,20 +51,25 @@ function ActionDropdown(props) {
 			{hideLabel ? null : <span>{label}</span>}
 		</span>
 	);
-
 	const style = link ? 'link' : bsStyle;
+	const onItemSelect = (object, event) => {
```
Comment (jmfrancois) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js]: if you don't need 'this' don't use arrow function, please use function instead.
https://github.com/Talend/ui/wiki/JavaScript-coding-style#lambda-aka-arrow-function

Comment (mtikhinskaya) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js]: Done

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://322.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://322.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://322.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://322.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://322.talend.surge.sh)

**Pull request merged**